app/controllers/ranking_records_controller.rb

```diff
@@ -0,0 +1,12 @@
+class RankingRecordsController < ApplicationController
+  def index
+    @records = Record.ranking_by(params[:sort], params[:page])
+    @offset = calculate_page_offset(@records)
+  end
+
+  private
+
+  def calculate_page_offset(records)
+    (records.current_page - 1) * records.limit_value
```

Comment:
以下のようにすれば引数なしでも良かったりしますかね？？

```suggestion
    (@records.current_page - 1) * @records.limit_value
```

Reply:
ありがとうございます！引数渡さずにインスタンス変数を使うようにします。